app/ide-desktop/lib/dashboard/src/appUtils.tsx

```diff
@@ -4,6 +4,10 @@
 // === Constants ===
 // =================
 
+// =============
+// === Paths ===
+// =============
+
```

Comment:
If this is sort of "subsection" of `Constants`, then: 
```suggestion
// === Paths ===
```